src/components/Input/SearchInput/SearchInput.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+import { ComponentProps } from 'react'
+import { InputWithTooltip } from '../InputWithTooltip'
+import { FaSearchIcon } from '../../Icon'
+
+type Props = Omit<ComponentProps<typeof InputWithTooltip>, 'tooltipMessage'> & {
```

Comment:
`<InputWithTooltip>` の `prefix` props を固定するならそれも `Omit` したほうが良さそう

```suggestion
type Props = Omit<ComponentProps<typeof InputWithTooltip>, 'tooltipMessage' | 'prefix' > & {
```

Reply:
修正しました! 78d5582